problem_set_1.py

```diff
@@ -42,13 +65,25 @@
 # fahrenheit_to_celsius function.
 # Absolute zero (0 K) is equivalent to −273.15 C.
 # 1 degree Kelvin equals 1 degree Celsius.
+def celsius_to_fahrenheit(temp):
+    return
+def fahrenheit_to_kelvin(temp): 
```

Comment:
If you want to use a function inside another function, you want to call it by name, `fahrenheit_to_kelvin(temp)`. Functions get defined using `def` once, and then you can call them any time after that.